cdhweb/people/models.py

```diff
@@ -493,6 +493,16 @@ class Profile(BasePage):
     parent_page_types = ["people.PeopleLandingPageArchived", "people.PeopleLandingPage"]
     subpage_types = []
 
+    promote_panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel("short_title"),
```

Comment:
Oh neat. The breadcrumbs needs to be updated to use this (if set) :)

Reply:
oh yeah thanks